pallene/types.lua

```diff
@@ -206,7 +206,9 @@ function types.consistent(t1, t2)
     return equivalent(t1, t2, true)
 end
 
+local inspect = require 'inspect'
 function types.tostring(t)
+    print(inspect(t))
```

Comment:
This inspect is printing a lot of junk when we run `busted`.

Reply:
I must have pushed it without commenting. Sorry. Will remove it.